### ISSUE TITLE
Stuck in dimmed screen mode

### ISSUE THREAD

**Issue by queimadus** [b-jesch/skin.estuary.modv2#138]
I'm not sure what's happening here but it only seems to occur with this skin enabled.
Basically, after being idles for a while the screen goes to what seems to be a dimmed mode and interacting with kodi does not undimm the screen. Navigating to the setting and changing the skin back to the default one makes the brightness go back to normal.

I've tried to look into the kodi log to see if there was any information there, but there's nothing.
I'd like to provide more information details but I'm not not sure what those might be.

**Comment by b-jesch** [comment]
While listening music You can dim the screen into black ("DIM"-Button on the navigation panel). Maybe this button is active? If not, please provide a debug log.

**Comment by queimadus** [comment]
The music dimmed button doesn't seem to be affected. I've also enabled debug logs and waited 1 min for the screensaver dimming to happen. I pressed a few keys to navigate around the menu, but unfortunately, nothing related appeared in the logs.

This is all happening in my Kodi installation for my [Vero 5](https://osmc.tv/vero/).  I've also installed this skin on my mac, and in there, it seems to work fine. After dimming, pressing any key undimms the screen. I pinpointed this issue to the skin, since it doesn't happen using the default skin, but it may be something related to the device.

This isn't much of a problem since I can just disable the screensaver dimming.
